README.md

```diff
@@ -1,9 +1,10 @@
-PyViz Topics Examples
----------------------
+# PyViz Topics Examples
 
-Domain-specific narrative examples using multiple PyViz projects
+Domain-specific narrative examples using multiple PyViz projects.
+Isolated fully described projects, runnable locally and deployable
+to Anaconda Enterprise.
```

Comment:
Each project is expected to have an author and a date (date created or date modified? not sure), and a canonical URL that can be sent around to send someone to this particular project.